modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -735,7 +748,8 @@
                               ;; tests expect the converted values.
                               :set-timezone             true
                               :expression-literals      true
-                              :database-routing         true}]
+                              :database-routing true
+                              :table-existence-checking true}]
```

Comment:
We've been trying to have namespaces for all new features. perhaps `:metadata/table-existence(-check)`